### PR TITLE
Add auto-refetch for Environments and Services

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-389)

- Add auto refetch for Environments and Services
- Fix status-chip, added CANCELED status  

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by RemiBonnet** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Add auto-refetch for Environments and Services - 2022-06-09T10:23:06Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/bd5a9c38-1f2f-4d64-9c0e-59bae531eede/applications)

File: libs/domains/environment/src/lib/slices/environments.slice.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const fetchEnvironments = createAsyncThunk<Environment[], { projectId: st
 export const fetchEnvironmentsStatus = createAsyncThunk<Status[], { projectId: string }>(
   'environments-status/fetch',
   async (data) => {
+    console.log(data)
```
Comment (WilliamTraoreee) [inline libs/domains/environment/src/lib/slices/environments.slice.ts:36]: You have a console.log here

**Review by WilliamTraoreee — COMMENTED** [review]
Good job @RemiBonnet ! :D

**Pull request merged**